internal/tools/orchestrator/scheduler/executor/plugins/k8s/deployment.go

```diff
@@ -691,6 +721,36 @@ func (k *Kubernetes) newDeployment(service *apistructs.Service, serviceGroup *ap
 	return deployment, nil
 }
 
+func setCoreErdaLabels(sg *apistructs.ServiceGroup, service *apistructs.Service, labels map[string]string) error {
+	if labels == nil {
+		return nil
+	}
+	if service != nil {
+		labels["core.erda.cloud/cluster-name"] = service.Labels["DICE_CLUSTER_NAME"]
```

Comment:
Maintain constants and label mapping relationships to avoid hardcoding them within the function.